common/projectconfig/projectconfig.go

```diff
@@ -32,16 +32,12 @@ type Config struct {
 	ExternalDirs  []string                    `toml:"external-dirs"`
 	Commands      Commands                    `toml:"commands"`
 	FTLMinVersion string                      `toml:"ftl-min-version"`
+	absModuleDirs []string
```

Comment:
This feels a bit gross, but I don't have any better ideas.

Reply:
I had exactly the same feeling. I'm also not 100% sure how we want the abs paths to be computed given that it's possible reference multiple `ftl-project.toml` files, but those files could be at different paths. Which, is what led me to this approach so that at least the `modules` will be relative to the `ftl-project.toml` file.

Definitely feels like this will need more thought in the future though.

Reply:
Yeah, and also the working directory of the process could change after construction, so we can't rely on dynamically making them absolute. It seems like the best solution.